lib/std/net.zig

```diff
@@ -157,7 +157,7 @@ pub const Address = extern union {
                     unreachable;
                 }
 
-                try std.fmt.format(out_stream, "{s}", .{&self.un.path});
+                try std.fmt.format(out_stream, "{s}", .{std.mem.sliceTo(std.meta.assumeSentinel(&self.un.path, 0), 0)});
```

Comment:
This `assumeSentinel()` call should not be needed, if it is I made a mistake in the implementation of `sliceTo()`
```suggestion
                try std.fmt.format(out_stream, "{s}", .{std.mem.sliceTo(&self.un.path, 0)});
```

Reply:
You're right, works fine without.